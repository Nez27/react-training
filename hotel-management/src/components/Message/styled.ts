import styled from 'styled-components';

const StyledMessage = styled.p`
  font-size: var(--fs-sm);
  text-align: center;
  padding: 20px;
`;

export { StyledMessage };
