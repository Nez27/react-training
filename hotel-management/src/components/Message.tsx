import styled from 'styled-components';

const StyledMessage = styled.p`
  font-size: var(--fs-sm);
  text-align: center;
  padding: 20px;
`;

const Message = ({ children }: { children: string }) => {
  return <StyledMessage>{children}</StyledMessage>;
};

export default Message;
