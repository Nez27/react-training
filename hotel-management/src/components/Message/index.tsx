// Styled
import { StyledMessage } from './styled';

const Message = ({ children }: { children: string }) => {
  return <StyledMessage>{children}</StyledMessage>;
};

export default Message;
