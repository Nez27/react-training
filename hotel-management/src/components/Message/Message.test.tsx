import renderer from 'react-test-renderer';

// Components
import Message from '.';

describe('Message testing snapshot', () => {
  const wrapper = renderer.create(<Message>This is a message!</Message>);

  test('render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
