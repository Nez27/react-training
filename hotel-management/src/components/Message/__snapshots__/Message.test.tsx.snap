// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Should render correctly 1`] = `
<p
  className="sc-bdnyFh fIKUZt"
>
  This is a message!
</p>
`;
